### PR TITLE
EL-4028-FixedHeader

### PR DESCRIPTION
**Pull request by Sdhilipkumar**
#### Checklist

* [x] The contents of this PR are mine to submit. No third party code or other material is being committed.
* [x] New third party dependencies (if any) are linked via `package.json`. Third party dependencies are licensed under one of the following: Apache, MIT, BSD, Mozilla Public License, Eclipse Public License, or Oracle Binary Code License.
* [x] The code in this PR does not contain export-restricted encryption algorithms.

#### Ticket / Issue

https://portal.digitalsafe.net/browse/EL-4028

#### Description of Proposed Changes
Added new class to achieve fixed header box-shadow.

#### Documentation CI URL

https://pages.github.houston.softwaregrp.net/sepg-docs-qa/UXAspects_CI_UXAspects_EL-4028-Fixed-Header

#### Downstream Build(s)

https://jenkins.swinfra.net/job/SEPG/job/caf/job/caf~ux-aspects-micro-focus~EL-4028-Fixed-Header~CI/

### CODE REVIEW

File: src/directives/fixed-header-table/fixed-header-table.directive.ts [hunk 1]
```diff
@@ -105,6 +109,9 @@ export class FixedHeaderTableDirective<T> implements OnInit, OnDestroy {
         if (delta < 1) {
             this.tablePaging.emit();
         }
+
+       // update the class based on the scroll position
+        scrollTop > 0 ? this._hasScrolled = true : this._hasScrolled = false;
```
Comment (donna-marijne) [inline src/directives/fixed-header-table/fixed-header-table.directive.ts:114]: This seems like a reasonable approach to me, but I would reverse this statement:
```ts
this._hasScrolled = (scrollTop > 0);
```

**Review by donna-marijne — CHANGES_REQUESTED** [review]

**Comment by donna-marijne** [comment]
Could you convert this to a draft until it is ready please?

**Comment by donna-marijne** [comment]
Please fix the PR text. You have a user script to populate it for you.

**Review by donna-marijne — APPROVED** [review]
Changes look good.

**Review by ashley-hunter — APPROVED** [review]
All good, thanks

**Pull request merged**